### ISSUE TITLE
Does this work with the GE Profile Indoor Smoker?

### ISSUE THREAD

**Issue by bfellner** [simbaja/ha_gehome#319]
Didn't see it listed and my smoker is due to arrive this Friday.  If you are willing to add it and need someone to test or gather information from then I am happy to help.

**Comment by simbaja** [comment]
It does not currently support the smoker, but if you run the [SDK](https://github.com/simbaja/gehome) application and open an issue over there, we can try to figure out the values needed to get information into HA.

**Comment by tlm4594** [comment]
Yes I would love to have this as well... Going to look at SDK now thank you very much!

**Comment by tlm4594** [comment]
> It does not currently support the smoker, but if you run the [SDK](https://github.com/simbaja/gehome) application and open an issue over there, we can try to figure out the values needed to get information into HA.

I wish I was more equipped to run the sdk.. is there anyway we can do a remote session? I'd be happy to pay you for your time to get my smokers value so we can expose it to HA.

**Comment by simbaja** [comment]
The new API reference helps identify all the codes needed (see #184 for links), but I don't have the time to create all the various sensors and controls at the moment.  Hopefully someone with some python experience can help out.  If there's a volunteer, I can help walk them through the steps needed to get it going.

**Comment by simbaja** [comment]
I'm also going to close in favor of #221 as it was an earlier request for this, let's discuss more over there.
